crates/pathfinder/src/sequencer.rs

```diff
@@ -556,9 +581,89 @@ mod tests {
     use assert_matches::assert_matches;
     use stark_hash::StarkHash;
 
-    /// Convenience wrapper
-    fn client() -> Client {
-        Client::new(Chain::Goerli).unwrap()
+    macro_rules! fixture {
+        ($path:literal) => {
+            include_str!(concat!("../fixtures/sequencer/", $path))
+        };
+    }
+
+    macro_rules! response {
+        ($path:literal) => {
+            (fixture!($path), 200)
+        };
+    }
```

Comment:
Some docs on what these do would be good.

Reply:
Docs missing here, definitely :+1: .